src/test/java/core/basesyntax/serviceimpl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.serviceimpl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exception.FileNotExistException;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String INCORRECT_PATH = "main/incorrectPath.csv";
+    private static final String PATH_TO_FILE_OK = "src/test/resources/Report.csv";
+    private static final String HEADER = "fruit,quantity";
+    private static final String SECOND_LINE = "banana,50";
+    private static final String EXPECTED_REPORT = HEADER
+            + System.lineSeparator() + SECOND_LINE;
+    private static List<String> listOk;
+    private static FileWriterServiceImpl fileWriterService;
+
```

Comment:
There might be another test to check the empty string input condition(but it's very optional, just an opinion)